charmhelpers/contrib/network/ovs/ovn.py

```diff
@@ -139,6 +139,18 @@ def is_cluster_leader(self):
         """
         return self.leader == 'self'
 
+    def to_yaml(self):
+        """Return yaml-serializable dict representation of this object.
+
+        This dict is guaranteed to be serializable with yaml.safe_dump().
```

Comment:
Please could you add a `:returns:` and `:rtype:` to the docstring for editor help/docs? e.g.

```
:returns: dictionary suitable for serialization by yaml.safe_dump()
:rytype: Dict[str, Any]
```

Reply:
Sure thing. Not sure how I missed this convention. Thanks.

Reply:
It's more a "code-gardening" type of thing.  Let's gradually try to improve the code and documentation each time we touch it :)